### PR TITLE
CdmrCoverage, remote and local; GribCollection fixes

### PR DESCRIPTION
**Pull request by JohnLCaron**

### CODE REVIEW

File: docs/website/netcdf-java/reference/stream/CdmrCommon.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
 CDM Remote is a web service implemented in the CDM library (client) and TDS (server), providing remote access to CDM datasets, using
-<<NcStream#,ncstream>> as the on-the-wire protocol.
+link:NcStream.adoc[ncstream] as the on-the-wire protocol. An experimental python client is under development.
```
Comment (cwardgar) [inline docs/website/netcdf-java/reference/stream/CdmrCommon.adoc]: This should remain a cross-reference. That way, when asciidoctor generates the HTML, the link will point to `NcStream.html`, not `NcStream.adoc`.

File: docs/website/netcdf-java/reference/stream/CdmrCommon.adoc [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-.  *<<CdmrFeatureGrid#,CDM Remote Feature for Coverage/Grid Data>>* provides coordinate subsetting on remote CDM Coverage (Grid) Feature Datasets, with similar functionality to WCS and
-Unidata's <<{tdsDocs}/NetcdfSubsetServiceReference#,NetCDF Subset Service>> (NCSS).
-.  *<<CdmrFeaturePoint#,CDM Remote Feature for Point Data>>* provides coordinate subsetting on remote CDM Point (DSG) Feature Datasets, with similar functionality to
-Unidata's <<{tdsDocs}/NetcdfSubsetServiceReference#,NetCDF Subset Service>> (NCSS).
+.  *CDM Remote Feature* provides coordinate subsetting on remote CDM Feature Datasets, with similar functionality to OGC's
+*Web Coverage Service* and the exact functionality of Unidata's *link:{tdsDocs}/NetcdfSubsetServiceReference.adoc[NetCDF Subset Service]* (NCSS).
+..  link:CdmrFeatureGrid.adoc[CDM Remote Feature for Coverage/Grid Data] provides coordinate subsetting on remote CDM Coverage (Grid) Feature Datasets
+..  link:CdmrFeaturePoint.adoc[CDM Remote Feature for Point Data] provides coordinate subsetting on remote CDM Point (DSG) Feature Datasets
```
Comment (cwardgar) [inline docs/website/netcdf-java/reference/stream/CdmrCommon.adoc]: All of these should be cross-references. Rule of thumb: we should never link to `.adoc` files.

Comment (JohnLCaron) [inline docs/website/netcdf-java/reference/stream/CdmrCommon.adoc]: what does a cross reference look like?

**Comment by cwardgar** [comment]
I'm waiting on some AsciiDoctor fixes and some `@Ignore` additions in `TestRdaReading` before I merge this in.

These mega PRs are really hard to review. Even GitHub thinks they're too big:
````Sorry, we could not display the entire diff because it was too big.````

**Comment by cwardgar** [comment]
In your latest commit, some of your xrefs need a trailing `#`. Here are the rules:

If you're linking to an anchor in the _current document_, your xref should look like:
````
Refer to <<Internal cross reference>>.
````
If you want to link to another document, but not a location within that document, you would use a trailing hash ('#'):
````
Go to <<another-document#>>
````
Finally, if you're linking to another location in another document, put the anchor name after the hash:
````
Go to <<another-document#example-section>>
````

[Source](https://github.com/asciidoctor/asciidoctor/issues/417#issuecomment-21982031). Yep, a GitHub comment. The official documentation is out-of-date. Kinda crappy.

We definitely need a "best practices for AsciiDoctor" document.

**Comment by cwardgar** [comment]
Looks like asciidoctor has some errors:
````
asciidoctor: WARNING: CdmrFeatureGrid.adoc: line 28: no callouts refer to list item 1
asciidoctor: WARNING: CdmrFeatureGrid.adoc: line 29: no callouts refer to list item 2
asciidoctor: WARNING: CdmrFeatureGrid.adoc: line 30: no callouts refer to list item 3
asciidoctor: WARNING: CdmrFeatureGrid.adoc: line 31: no callouts refer to list item 4
asciidoctor: WARNING: arams.adoc: line -1: no callouts refer to list item 5
````

I'm considering merging this in and tracking those down, since I have to investigate the `CacheAggregationsSpec` failure anyway. Otherwise, this PR looks good.

**Comment by JohnLCaron** [comment]
those are the callouts in the table that dont work. I thought Id leave them
in in case a workaround shows up, but i can remove them.

**Comment by JohnLCaron** [comment]
fixed now (i think)

**Pull request merged**